frontend/src/components/Place/PlaceItem.tsx

```
@@ -1,38 +1,38 @@
+    moveToLocation(location.latitude, location.longitude);
   };
 
   return (
     <article
       role="button"
       tabIndex={0}
       onClick={onPlaceClick}
-      onKeyDown={onKeyDown}
-      className={`flex items-center rounded-md border-[1px] ${place.id === activePlace.id && !isDetailPage ? 'border-1 border-c_bg_blue' : 'border-c_border_gray'} p-4 ${isDetailPage ? 'cursor-default' : ''}`}
+      className={`flex items-center rounded-md border-[1px] ${place.id === activePlace.id && !isDetailPage ? 'border-1 border-c_bg_blue' : 'border-c_border_gray'} p-4`}
     >
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)